javascripts/discourse/components/new-pm-dropdown-button.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if this.currentUser.can_send_private_messages}}
+  <DButton
+      @class="btn-default btn-icon-text new-pm-dropdown-button"
+      @icon="envelope"
+      @label="user.new_private_message"
+      @title="user.new_private_message"
```

Comment:
Nitpicky, but I don't see the value for a title identical to a label... I guess here we sometimes hide the label via CSS, so that kinda makes sense.